README.md

```diff
@@ -490,4 +492,4 @@ Here's a complete example showing all available options:
     api-url: https://api.devicecloud.dev
     config: path/to/maestro-config.yaml
     runner-type: default   # Experimental: m1|m4
-```
\ No newline at end of file
+```
```

Comment:
I'm not sure what's happening here - I tried twice to edit without this, and each time it showed up. It doesn't seem to have any effect, though.